### PR TITLE
simplify usage examples by placing in single notebook

### PR DESCRIPTION
**Pull request by NLaws**
Consolidated the multi-scenario and single scenario examples into a single notebook

### CODE REVIEW

File: src/multi_site_inputs_parser.py [hunk 1]
```diff
@@ -131,9 +128,18 @@ def multi_site_csv_parser(path_to_csv, api_url, API_KEY, n_sites=None):
...
-        add_load_profile_inputs(site_inputs, posts[-1], path_to_load_files=os.path.join(path_to_csv[:path_to_csv.index(os.sep)], 'load_profiles'))
+        path_to_load_files = "./load_profiles"
+        add_load_profile_inputs(site_inputs, posts[-1], path_to_load_files=path_to_load_files)
 
         posts[i]['Scenario']['Site']['Wind'] = {'max_kw': 0}  # hack for Wind not in help endpoint
 
     return posts
 
+
+test = False
```
Comment (t-kwasnik) [inline src/multi_site_inputs_parser.py:139]: @NLaws is this intentional?

File: src/parse_api_responses_to_csv.py [hunk 2]
```diff
@@ -12,8 +12,8 @@ def get_nested_output(key, resp, obj=None):
         return resp['outputs']['Scenario']['Site'][obj][val]
 
     else:
-        site = resp['outputs']['Scenario']['Site']
-
+        site = {k: v for k, v in resp['outputs']['Scenario']['Site'].items() if k.isupper()}
```
Comment (t-kwasnik) [inline src/parse_api_responses_to_csv.py:15]: @NLaws what is this trying to do? Should it be k[0].isupper()?

File: src/post_and_poll.py [hunk 3]
```diff
@@ -67,6 +67,6 @@ def get_run_uuid(post, API_KEY, api_url):
     run_uuid = "my_run_id"
     file_name = "my_results.json"
     api_url = 'https://developer.nrel.gov/api/reopt/v1'
-    from keys import API_KEY
+    API_KEY = "DEMO_KEY"
```
Comment (t-kwasnik) [inline src/post_and_poll.py:70]: @NLaws do we want DEMO_KEY in here? That's a valid key that will return responses. Should we load it from the environment?

**Review by t-kwasnik — COMMENTED** [review]
@NLaws left a few comments, let me know if these are really non-issues and I can approve

File: src/multi_site_inputs_parser.py (hunk 1 above)
Comment (NLaws) [reply src/multi_site_inputs_parser.py:139]: yes

File: src/parse_api_responses_to_csv.py (hunk 2 above)
Comment (NLaws) [reply src/parse_api_responses_to_csv.py:15]: It filters out all the Site attributes that are not capitalized, such as `latitude`.  I think that you're right. I'll test it locally.

File: src/post_and_poll.py (hunk 3 above)
Comment (NLaws) [reply src/post_and_poll.py:70]: yes this is to help people get started. we could use the keys.py but I felt like that added another layer of abstraction. Just trying to keep it as simple as possible.

File: src/parse_api_responses_to_csv.py (hunk 2 above)
Comment (NLaws) [reply src/parse_api_responses_to_csv.py]: fixed with https://github.com/NREL/REopt-API-Analysis/pull/7/commits/fafa0a301cca8a8172dfc143bc0997f658d102b5

**Comment by NLaws** [comment]
@t-kwasnik ^^^

**Review by t-kwasnik — APPROVED** [review]

**Pull request merged**